--- dnssec-validator.orig/Makefile.layout
+++ dnssec-validator/Makefile.layout
@@ -28,24 +28,4 @@
 
 BASEDIR := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 
-LIBS_ARCHIVE_DIR = libs-archives
-LIBS_SRC_DIR = libs-src
-#LIBS_BUILT_DIR ?= libs-built-$(OSNAME_LC)-$(HWARCH)
-LIBS_BUILT_DIR ?= libs-built
-
-OPENSSL_TARBALL = $(BASEDIR)/$(LIBS_ARCHIVE_DIR)/openssl.tar.gz
-LDNS_TARBALL = $(BASEDIR)/$(LIBS_ARCHIVE_DIR)/ldns.tar.gz
-UNBOUND_TARBALL = $(BASEDIR)/$(LIBS_ARCHIVE_DIR)/unbound.tar.gz
-
-OPENSSL_DIR = $(BASEDIR)/$(LIBS_SRC_DIR)/openssl-1.0.1h
-LDNS_DIR = $(BASEDIR)/$(LIBS_SRC_DIR)/ldns-1.6.17
-UNBOUND_DIR = $(BASEDIR)/$(LIBS_SRC_DIR)/unbound-1.4.22
-
-OPENSSL_LIB = $(BASEDIR)/$(LIBS_BUILT_DIR)/openssl
-LDNS_LIB = $(BASEDIR)/$(LIBS_BUILT_DIR)/ldns
-UNBOUND_LIB = $(BASEDIR)/$(LIBS_BUILT_DIR)/unbound
-
-FIREBREATH_DIR = $(BASEDIR)/FireBreath
-FIREBREATH_TAG = firebreath-1.6.0
-
-PLUGIN_BUILD_DIR = $(FIREBREATH_DIR)/build
+PLUGIN_BUILD_DIR = build
--- dnssec-validator.orig/Makefile.main
+++ dnssec-validator/Makefile.main
@@ -1,6 +1,6 @@
-.PHONY: all clean download-git-firebreath scour plugin
+.PHONY: all clean scour plugin
 
-all: download-git-firebreath plugin
+all: plugin
 
 CFLAGS=-fPIC
 export CFLAGS
@@ -65,18 +65,8 @@ endif
 ## You need to run 'make prepmake' after changing.
 PLUGIN_CONFIGURATION ?= MinSizeRel
 
-download-git-firebreath:
-	rm -rf $(FIREBREATH_DIR)
-	git clone https://github.com/firebreath/FireBreath.git $(FIREBREATH_DIR)
-	cd $(FIREBREATH_DIR); git reset --hard 66af0df36be98e9fcf7ee196fe5c275e54752ae8; cd .. # 2014-06-24 17:48:14
-	#git reset --hard origin/master
-	cd ..
-	git submodule update --init --recursive
-	#(cd $(FIREBREATH_DIR) && git checkout $(FIREBREATH_TAG))
-
 # Build the plug-in.
 plugin:
-	CFLAGS="$(CFLAGS)" $(MAKE_CMD) -f Makefile.static_libs OPENSSL_ARGS="$(OPENSSL_ARGS)" CONFIGURE_ARGS="$(CONFIGURE_ARGS)" OSNAME=$(OSNAME) OSNAME_LC=$(OSNAME_LC) HWARCH=$(HWARCH) MAKE_CMD=$(MAKE_CMD) STATIC_LINKING=$(STATIC_LINKING)
 	CFLAGS="$(CFLAGS)" $(MAKE_CMD) -f $(MAKE_OS) OSNAME=$(OSNAME) OSNAME_LC=$(OSNAME_LC) HWARCH=$(HWARCH) MAKE_CMD=$(MAKE_CMD) INCLUDE_LOC=$(INCLUDE_LOC) LIBRARY_LOC=$(LIBRARY_LOC) STATIC_LINKING=$(STATIC_LINKING) $(CORE_TGT)
 	CFLAGS="$(CFLAGS)" $(MAKE_CMD) -f $(MAKE_OS) OSNAME=$(OSNAME) OSNAME_LC=$(OSNAME_LC) HWARCH=$(HWARCH) MAKE_CMD=$(MAKE_CMD) INCLUDE_LOC=$(INCLUDE_LOC) LIBRARY_LOC=$(LIBRARY_LOC) $(XPI_TGT)
 
--- dnssec-validator.orig/Makefile.unix
+++ dnssec-validator/Makefile.unix
@@ -60,22 +60,6 @@ sys_$(OSNAME_LC)_x86: npapi_x86 js-ctype
 
 sys_$(OSNAME_LC)_x64: npapi_x64 js-ctypes_x64
 
-npapi_x86:
-	@echo '### ...Creating NPAPI plugin library for $(OSNAME) x86... ###'
-	rm -rf $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so $(PLUGIN_FB_S)
-	mkdir $(PLUGIN_NAME_LIB) || echo "$(PLUGIN_NAME_LIB) already exists."
-	cd FireBreath && mkdir projects && cd ..
-	cp -r plugin-source/* $(PLUGIN_FB_S)
-	$(SHELL_CMD) ./FireBreath/prepmake.sh $(PLUGIN_FB_S) $(PLUGIN_FB_B) -DSTATIC_LINKING=$(STATIC_LINKING) -DLIBS_BUILT_DIR=$(LIBS_BUILT_DIR) -DINCLUDE_LOC=$(INCLUDE_LOC) -DLIBRARY_LOC=$(LIBRARY_LOC) -DCMAKE_VERBOSE_MAKEFILE=1 -DCMAKE_C_FLAGS=-m32 -DCMAKE_CXX_FLAGS=-m32 -DCMAKE_BUILD_TYPE=MinSizeRel
-	$(MAKE_CMD) -C $(PLUGIN_FB_B)
-	cp $(PLUGIN_FB_B)/bin/$(NPAPI_PLUGIN_NAME_DNSSEC)/np$(NPAPI_PLUGIN_NAME_DNSSEC).so $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so
-	cp $(PLUGIN_FB_B)/bin/$(NPAPI_PLUGIN_NAME_TLSA)/np$(NPAPI_PLUGIN_NAME_TLSA).so $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so
-	strip $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so
-	strip $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so
-	rm -rf $(PLUGIN_FB_B)
-	rm -rf $(PLUGIN_FB_S)
-	@echo '### ...NPAPI plugin x86 $(OSNAME) ... Done. ###'
-
 js-ctypes_x86:
 	@echo '### ...Creating js-ctypes plugin library for $(OSNAME) x86... ###'
 	rm -rf $(PLUGIN_NAME_LIB)/$(JSCTYPES_PLUGIN_NAME_DNSSEC)-x86.so  $(PLUGIN_NAME_LIB)/$(JSCTYPES_PLUGIN_NAME_DANE)-x86.so
@@ -89,21 +73,6 @@ js-ctypes_x86:
 	@echo '### ...js-ctypes plugin x86 $(OSNAME) ... Done. ###'
 
 xpi_x86:
-	@echo '### Creating NPAPI package for Firefox $(OSNAME) x86... ###'
-	rm -rf $(ADDON_PATH_FF)/npapi_built
-	mkdir $(ADDON_PATH_FF)/npapi_built
-	mkdir $(ADDON_PATH_FF)/npapi_built/plugins
-	cp -r $(ADDON_PATH_FF)/npapi/* $(ADDON_PATH_FF)/npapi_built
-	cp -r $(ADDON_PATH_FF)/common/* $(ADDON_PATH_FF)/npapi_built
-	chmod 777 $(ADDON_PATH_FF)/npapi_built/build.sh
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so $(ADDON_PATH_FF)/npapi_built/plugins
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so $(ADDON_PATH_FF)/npapi_built/plugins
-	sed -e 's/<em:version><\/em:version>/<em:version>$(EXTENSION_VERSION)<\/em:version>/g' -e 's/<em:targetPlatform><\/em:targetPlatform>/<em:targetPlatform>$(OSNAME)<\/em:targetPlatform>/g' $(ADDON_PATH_FF)/npapi_built/install.rdf.template > $(ADDON_PATH_FF)/npapi_built/install.rdf
-	sed -e 's/xversionx/$(EXTENSION_VERSION)/g' $(ADDON_PATH_FF)/npapi_built/about.template > $(ADDON_PATH_FF)/npapi_built/content/about.xul
-	if [ ! -d "$(PLUGIN_ADD_ON)" ]; then mkdir $(PLUGIN_ADD_ON); fi
-	cd $(ADDON_PATH_FF)/npapi_built && $(SHELL_CMD) ./build.sh && mv dnssec.xpi ../../../$(PLUGIN_ADD_ON)/MF-dnssec-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86-npapi.xpi
-	@echo '### DONE... ###'
-	@echo '--------------------------------------------------'
 	@echo '### Creating js-ctypes package for Firefox $(OSNAME) x86... ###'
 	rm -rf $(ADDON_PATH_FF)/js-ctypes_built
 	mkdir $(ADDON_PATH_FF)/js-ctypes_built
@@ -120,51 +89,6 @@ xpi_x86:
 	if [ ! -d "$(PLUGIN_ADD_ON)" ]; then mkdir $(PLUGIN_ADD_ON); fi
 	cd $(ADDON_PATH_FF)/js-ctypes_built && $(SHELL_CMD) ./build.sh && mv dnssec.xpi ../../../$(PLUGIN_ADD_ON)/MF-dnssec-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86-js-ctypes.xpi
 	@echo '### DONE... ###'
-	@echo '--------------------------------------------------'
-	@echo '### Creating package for Chrome $(OSNAME) x86... ###'
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so $(ADDON_PATH_CR_DNSSEC)
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so $(ADDON_PATH_CR_TLSA)
-	sed -e 's/dnssecplugin/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_CR_DNSSEC)/manifest.json.template > $(ADDON_PATH_CR_DNSSEC)/manifest.json
-	sed -e 's/tlsaplugin/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_CR_TLSA)/manifest.json.template > $(ADDON_PATH_CR_TLSA)/manifest.json
-	cd add-on && tar -czf CR-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz chrome2dnssec && mv CR-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz ../$(PLUGIN_ADD_ON)
-	cd add-on && tar -czf CR-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz chrome2tlsa && mv CR-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz ../$(PLUGIN_ADD_ON)
-	cd $(ADDON_PATH_CR_DNSSEC) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so
-	cd $(ADDON_PATH_CR_TLSA) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so
-	@echo '### DONE... ###'
-	@echo '--------------------------------------------------'
-	@echo '### Creating package for Opera $(OSNAME) x86... ###'
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so $(ADDON_PATH_OP_DNSSEC)
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so $(ADDON_PATH_OP_TLSA)
-	sed -e 's/dnssecplugin/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_OP_DNSSEC)/manifest.json.template > $(ADDON_PATH_OP_DNSSEC)/manifest.json
-	sed -e 's/tlsaplugin/np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_OP_TLSA)/manifest.json.template > $(ADDON_PATH_OP_TLSA)/manifest.json
-	cd add-on && tar -czf OP-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz opera2dnssec && mv OP-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz ../$(PLUGIN_ADD_ON)
-	cd add-on && tar -czf OP-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz opera2tlsa && mv OP-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x86.tar.gz ../$(PLUGIN_ADD_ON)
-	cd $(ADDON_PATH_OP_DNSSEC) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_DNSSEC)_x86.so
-	cd $(ADDON_PATH_OP_TLSA) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_TLSA)_x86.so
-	@echo '### DONE... ###'
-	@echo
-	@echo
-	@echo
-	@echo
-	@echo
-	@echo
-	@echo
-
-npapi_x64:
-	@echo '### ...Creating NPAPI plugin library for $(OSNAME) x64... ###'
-	rm -rf $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so $(PLUGIN_FB_S)
-	mkdir $(PLUGIN_NAME_LIB) || echo "$(PLUGIN_NAME_LIB) already exists."
-	cd FireBreath && mkdir projects && cd ..
-	cp -r plugin-source/* $(PLUGIN_FB_S)
-	$(SHELL_CMD) ./FireBreath/prepmake.sh $(PLUGIN_FB_S) $(PLUGIN_FB_B) -DSTATIC_LINKING=$(STATIC_LINKING) -DLIBS_BUILT_DIR=$(LIBS_BUILT_DIR) -DINCLUDE_LOC=$(INCLUDE_LOC) -DLIBRARY_LOC=$(LIBRARY_LOC) -DCMAKE_VERBOSE_MAKEFILE=1 -DCMAKE_C_FLAGS=-m64 -DCMAKE_CXX_FLAGS=-m64 -DCMAKE_BUILD_TYPE=MinSizeRel
-	$(MAKE_CMD) -C $(PLUGIN_FB_B)
-	cp $(PLUGIN_FB_B)/bin/$(NPAPI_PLUGIN_NAME_DNSSEC)/np$(NPAPI_PLUGIN_NAME_DNSSEC).so $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so
-	cp $(PLUGIN_FB_B)/bin/$(NPAPI_PLUGIN_NAME_TLSA)/np$(NPAPI_PLUGIN_NAME_TLSA).so $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so
-	strip $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so
-	strip $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so
-	rm -rf $(PLUGIN_FB_B)
-	rm -rf $(PLUGIN_FB_S)
-	@echo '### ...NPAPI plugin x64 $(OSNAME) ... Done. ###'
 
 js-ctypes_x64:
 	@echo '### ...Creating js-ctypes plugin library for $(OSNAME) x64... ###'
@@ -180,21 +104,6 @@ js-ctypes_x64:
 	@echo '### ...js-ctypes plugin x64 $(OSNAME) ... Done. ###'
 
 xpi_x64:
-	@echo '### Creating NPAPI package for Firefox $(OSNAME) x64... ###'
-	rm -rf $(ADDON_PATH_FF)/npapi_built
-	mkdir $(ADDON_PATH_FF)/npapi_built
-	mkdir $(ADDON_PATH_FF)/npapi_built/plugins
-	cp -r $(ADDON_PATH_FF)/npapi/* $(ADDON_PATH_FF)/npapi_built
-	cp -r $(ADDON_PATH_FF)/common/* $(ADDON_PATH_FF)/npapi_built
-	chmod 777 $(ADDON_PATH_FF)/npapi_built/build.sh
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so $(ADDON_PATH_FF)/npapi_built/plugins
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so $(ADDON_PATH_FF)/npapi_built/plugins
-	sed -e 's/<em:version><\/em:version>/<em:version>$(EXTENSION_VERSION)<\/em:version>/g' -e 's/<em:targetPlatform><\/em:targetPlatform>/<em:targetPlatform>$(OSNAME)<\/em:targetPlatform>/g' $(ADDON_PATH_FF)/npapi_built/install.rdf.template > $(ADDON_PATH_FF)/npapi_built/install.rdf
-	sed -e 's/xversionx/$(EXTENSION_VERSION)/g' $(ADDON_PATH_FF)/npapi_built/about.template > $(ADDON_PATH_FF)/npapi_built/content/about.xul
-	if [ ! -d "$(PLUGIN_ADD_ON)" ]; then mkdir $(PLUGIN_ADD_ON); fi
-	cd $(ADDON_PATH_FF)/npapi_built && $(SHELL_CMD) ./build.sh && mv dnssec.xpi ../../../$(PLUGIN_ADD_ON)/MF-dnssec-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64-npapi.xpi
-	@echo '### DONE... ###'
-	@echo '--------------------------------------------------'
 	@echo '### Creating js-ctypes package for Firefox $(OSNAME) x64... ###'
 	rm -rf $(ADDON_PATH_FF)/js-ctypes_built
 	mkdir $(ADDON_PATH_FF)/js-ctypes_built
@@ -212,34 +121,6 @@ xpi_x64:
 	cd $(ADDON_PATH_FF)/js-ctypes_built && $(SHELL_CMD) ./build.sh && mv dnssec.xpi ../../../$(PLUGIN_ADD_ON)/MF-dnssec-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64-js-ctypes.xpi
 	@echo '### DONE... ###'
 	@echo '--------------------------------------------------'
-	@echo '### Creating package for Chrome $(OSNAME) x64... ###'
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so $(ADDON_PATH_CR_DNSSEC)
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so $(ADDON_PATH_CR_TLSA)
-	sed -e 's/dnssecplugin/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_CR_DNSSEC)/manifest.json.template > $(ADDON_PATH_CR_DNSSEC)/manifest.json
-	sed -e 's/tlsaplugin/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_CR_TLSA)/manifest.json.template > $(ADDON_PATH_CR_TLSA)/manifest.json
-	cd add-on && tar -czf CR-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz chrome2dnssec && mv CR-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz ../$(PLUGIN_ADD_ON)
-	cd add-on && tar -czf CR-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz chrome2tlsa && mv CR-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz ../$(PLUGIN_ADD_ON)
-	cd $(ADDON_PATH_CR_DNSSEC) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so
-	cd $(ADDON_PATH_CR_TLSA) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so
-	@echo '### DONE... ###'
-	@echo '--------------------------------------------------'
-	@echo '### Creating package for Opera $(OSNAME) x64... ###'
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so $(ADDON_PATH_OP_DNSSEC)
-	cp $(PLUGIN_NAME_LIB)/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so $(ADDON_PATH_OP_TLSA)
-	sed -e 's/dnssecplugin/np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_OP_DNSSEC)/manifest.json.template > $(ADDON_PATH_OP_DNSSEC)/manifest.json
-	sed -e 's/tlsaplugin/np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so/g' -e 's/version-of-add-on/$(EXTENSION_VERSION_CR)/g' $(ADDON_PATH_OP_TLSA)/manifest.json.template > $(ADDON_PATH_OP_TLSA)/manifest.json
-	cd add-on && tar -czf OP-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz opera2dnssec && mv OP-dnssec_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz ../$(PLUGIN_ADD_ON)
-	cd add-on && tar -czf OP-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz opera2tlsa && mv OP-tlsa_validator-$(EXTENSION_VERSION)-$(OSNAME_LC)-x64.tar.gz ../$(PLUGIN_ADD_ON)
-	cd $(ADDON_PATH_OP_DNSSEC) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_DNSSEC)_x64.so
-	cd $(ADDON_PATH_OP_TLSA) && rm -rf manifest.json np$(NPAPI_PLUGIN_NAME_TLSA)_x64.so
-	@echo '### DONE... ###'
-	@echo
-	@echo
-	@echo
-	@echo
-	@echo
-	@echo
-	@echo
 
 clean:
 	rm -rf $(PLUGIN_NAME_LIB) $(PLUGIN_FB_S)
